controllers/yawol-controller/loadbalancer/loadbalancer_controller_test.go

```diff
@@ -703,29 +703,26 @@ var _ = Describe("loadbalancer controller", Serial, Ordered, func() {
 		BeforeEach(func() {
 			mockClient.GroupClientObj = &testing.CallbackGroupClient{
 				ListFunc: func(ctx context.Context, opts groups.ListOpts) ([]groups.SecGroup, error) {
-					return []groups.SecGroup{}, gophercloud.ErrDefault401{
-						ErrUnexpectedResponseCode: gophercloud.ErrUnexpectedResponseCode{
-							Body: []byte("Auth failed"),
-						},
+					return []groups.SecGroup{}, gophercloud.ErrUnexpectedResponseCode{
+						// Actual: 401,
```

Comment:
From what I could find, the `gophercloud.ErrDefaultXXX` structs were just a wrapper around `gophercloud.ErrUnexpectedResponseCode`, without actually setting anything, so `Actual` actually doesn't need to be set.